### ISSUE TITLE
4.23.4 has a runtime dependency on gtest

### ISSUE THREAD

**Issue by lidavidm** [conda-forge/libprotobuf-feedstock#186]
### Solution to issue cannot be found in the documentation.

- [X] I checked the documentation.

### Issue

Installing `libprotobuf` also installs `gtest`, which I would think is not a runtime dependency (I may be wrong!) 4.23.3 doesn't appear to have this issue.

To wit:
```
$ mamba create -n tempasdf2 libprotobuf=4.23.4
<snip...>
Looking for: ['libprotobuf=4.23.4']

conda-forge/linux-64                                        Using cache
conda-forge/noarch                                          Using cache
Transaction

  Prefix: /home/lidavidm/miniforge3/envs/tempasdf2

  Updating specs:

   - libprotobuf=4.23.4


  Package             Version  Build             Channel                    Size
──────────────────────────────────────────────────────────────────────────────────
  Install:
──────────────────────────────────────────────────────────────────────────────────

  + _libgcc_mutex         0.1  conda_forge       conda-forge/linux-64     Cached
  + _openmp_mutex         4.5  2_gnu             conda-forge/linux-64     Cached
  + gtest              1.14.0  h00ab1b0_1        conda-forge/linux-64     Cached
  + libabseil      20230802.0  cxx17_h59595ed_3  conda-forge/linux-64     Cached
  + libgcc-ng          13.2.0  h807b86a_0        conda-forge/linux-64     Cached
  + libgomp            13.2.0  h807b86a_0        conda-forge/linux-64     Cached
  + libprotobuf        4.23.4  hf27288f_5        conda-forge/linux-64     Cached
  + libstdcxx-ng       13.2.0  h7e041cc_0        conda-forge/linux-64     Cached
  + libzlib            1.2.13  hd590300_5        conda-forge/linux-64     Cached

$ jq .depends $CONDA_PREFIX/conda-meta/libprotobuf-4.23.4-hf27288f_5.json
[
  "libgcc-ng >=12",
  "libstdcxx-ng >=12",
  "libzlib >=1.2.13,<1.3.0a0",
  "libabseil * cxx17*",
  "gtest >=1.14.0,<1.14.1.0a0",
  "libabseil >=20230802.0,<20230803.0a0"
]
$ cat info/index.json
{
  "arch": "x86_64",
  "build": "hd1fb520_1",
  "build_number": 1,
  "depends": [
    "libabseil * cxx17*",
    "libabseil >=20230125.3,<20230126.0a0",
    "libgcc-ng >=12",
    "libstdcxx-ng >=12",
    "libzlib >=1.2.13,<1.3.0a0"
  ],
  "license": "BSD-3-Clause",
  "license_family": "BSD",
  "name": "libprotobuf",
  "platform": "linux",
  "subdir": "linux-64",
  "timestamp": 1693582252190,
  "version": "4.23.3"
}
```

### Installed packages

```shell
# packages in environment at /home/lidavidm/miniforge3/envs/tempasdf:
#
# Name                    Version                   Build  Channel
_libgcc_mutex             0.1                 conda_forge    conda-forge
_openmp_mutex             4.5                       2_gnu    conda-forge
gtest                     1.14.0               h00ab1b0_1    conda-forge
libabseil                 20230802.0      cxx17_h59595ed_3    conda-forge
libgcc-ng                 13.2.0               h807b86a_0    conda-forge
libgomp                   13.2.0               h807b86a_0    conda-forge
libprotobuf               4.23.4               hf27288f_5    conda-forge
libstdcxx-ng              13.2.0               h7e041cc_0    conda-forge
libzlib                   1.2.13               hd590300_5    conda-forge
```

### Environment info

```shell
active environment : tempasdf
    active env location : /home/lidavidm/miniforge3/envs/tempasdf
            shell level : 2
       user config file : /home/lidavidm/.condarc
 populated config files : /home/lidavidm/miniforge3/.condarc
                          /home/lidavidm/.condarc
          conda version : 22.11.1
    conda-build version : not installed
         python version : 3.10.9.final.0
       virtual packages : __archspec=1=x86_64
                          __cuda=12.1=0
                          __glibc=2.31=0
                          __linux=5.4.0=0
                          __unix=0=0
       base environment : /home/lidavidm/miniforge3  (writable)
      conda av data dir : /home/lidavidm/miniforge3/etc/conda
  conda av metadata url : None
           channel URLs : https://conda.anaconda.org/conda-forge/linux-64
                          https://conda.anaconda.org/conda-forge/noarch
          package cache : /home/lidavidm/miniforge3/pkgs
                          /home/lidavidm/.conda/pkgs
       envs directories : /home/lidavidm/miniforge3/envs
                          /home/lidavidm/.conda/envs
               platform : linux-64
             user-agent : conda/22.11.1 requests/2.28.2 CPython/3.10.9 Linux/5.4.0-162-generic ubuntu/20.04.6 glibc/2.31
                UID:GID : 1001:1001
             netrc file : /home/lidavidm/.netrc
           offline mode : False
```

**Comment by h-vetinari** [comment]
Thanks for the report. Fix is in #187. gtest had gained a run-export in the meantime, which we now need to ignore.

**Comment by lidavidm** [comment]
great, thanks!
